addons/medical_feedback/RscInGameUI.hpp

```diff
@@ -0,0 +1,25 @@
+
+class RscPictureKeepAspect;
+
+#define W_GRID(num) (num * (((safezoneW / safezoneH) min 1.2) / 40))
+#define H_GRID(num) (num * ((((safezoneW / safezoneH) min 1.2) / 1.2) / 25))
+
+#define X_STANCE_BG ((safezoneX + safezoneW) - W_GRID(3.7) - W_GRID(0.5))
+#define Y_STANCE_BG (safezoneY + H_GRID(0.8))
+#define W_STANCE_BG W_GRID(3.7)
+#define H_STANCE_BG H_GRID(3.7)
+
+class RscInGameUI {
+    class RscStanceInfo {
+        controls[] += { QGVAR(BloodVolumeInfoIndicator) };
+        class GVAR(BloodVolumeInfoIndicator): RscPictureKeepAspect {
+            idc = 100205;
```

Comment:
Where does this come from? Seems unused. If so, just use -1, which should already be inherited.

Reply:
Removed idc and text definition